tests/behaviour/inspectcli.feature

```diff
@@ -319,7 +319,7 @@ Feature: Behaviour testing of csvcubed inspect.
         - The data cube has the following value counts:
             - Value counts broken-down by measure and unit (of measure):
                        Measure                                          Unit  Count
```

Comment:
I take it this is the old-style CSV-W we're using to ensure the inspect command functionality is compatible with it? Could you add a comment or amend the scenario name to make it clear that the scenario tests this outcome too?